modules/administration-guide/pages/removing-user-data.adoc

```diff
@@ -8,141 +8,48 @@
-  --type merge \
-  -p '{ "spec": { "server": {"customCheProperties": {"pass:[CHE_SYSTEM_SUPER__PRIVILEGED__MODE]": "true"} } }}' \
-  -n __{prod-namespace}__
-----
-+
-[subs="+quotes,macros,attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} \
-  --type merge \
-  -p '{ "spec": { "server": {"customCheProperties": {"pass:[CHE_SYSTEM_ADMIN__NAME]": "__<admin-name>__"} } }}' \
-  -n __{prod-namespace}__
-----
-+
-[NOTE]
-====
-All {prod-short} system permissions are granted to the administrative user. To configure the administrative user, use the `CHE_SYSTEM_ADMIN__NAME` property. The default value is `admin`. The system permissions are granted when the {prod-short} server starts. If the user record is not in the {prod-short} user database, the permissions are granted after the first login of the user.
-
-.Authorization token privileges:
+* A __<token>__ authorization token for a user with `admin` privileges. See xref:configuring-the-administrative-user.adoc[].
 
-* `admin` - Can delete all personal data of all users
-* `user` - Can delete only the data related to the user
-====
-
-* A user or an administrator is logged in the OpenShift cluster with deployed {prod-short}.
-
-* A user ID is obtained. Get the user ID using the commands below:
+.Procedure
 
-** For the current user:
+. Get the __<username>__ user __<id>__:
 +
 [subs="+quotes,macros,attributes"]
 ----
 $ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
-  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user'
-----
- 
-** To find a user by name: 
-+
-[subs="+quotes,macros,attributes"]
-----
-$ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
+  --header 'Authorization: Bearer __<token>__' \
   'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
 ----
 
-.Example of obtaining a user ID
-
-This example uses `vparfono` as a local user name.
-
-====
-[subs="+quotes,macros,attributes"]
-----
-$ curl -X GET \
-  --header 'Authorization: Bearer __<user-token>__' \
-  'https://che-vp-che.apps.che-dev.x6e0.p1.openshiftapps.com/api/user/find?name=vparfono'
-----
-
-The user ID is at the bottom of the curl command output.
-
-[source,json]
-----
-{
- "name": "vparfono",
- "links": [
-   {
-.
-.
-.
-   }
- ],
- "email": "vparfono@redhat.com",
- "id": "921b6f33-2657-407e-93a6-fb14cf2329ce"
-}
-----
-====
-
-.Procedure
-
-* Remove the data using the API:
+. Remove the user __<id>__ data:
 +
 [subs="+quotes,macros,attributes"]
 ----
 $ curl -i -X DELETE \
-  --header 'Authorization: Bearer __<user-token>__' \
-  pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/__<user-id>__
+  --header 'Authorization: Bearer __<token>__' \
+  pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/__<id>__
 ----
 
-[NOTE]
-====
-The procedure can be also executed using the Swagger UI available on pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger
-====
+. Optionally, remove the {orch-name} resources bound to the user.
```

Comment:
Removing Kubernetes resources is mentioned as optional. But git credentials, email addresses and other user related information are stored in secrets and configmaps, and persistent volume may contain user data too. Shouldn't this be the most important part of the user data removal?

Reply:
yes, you can add the info about removing namespace completely to delete all k8s resources bound to the use

Reply:
Do you have a ready-to-use kubectl one-liner that we could use?

Using OpenShift console, I see that from the user page, we can list RoleBindings assigned to the user, and then the namespace to delete. I am searching how to do the same using CLI.

After removing the user using Che server API, and removing the user namespace, we are sure we deleted all data?

Reply:
We can rely on the default namespace name.